SourceCode2Pdf/PythonScriptsExecution/PythonScriptsExecution.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using IronPython.Hosting;
+using Microsoft.Scripting.Hosting;
+
+namespace SourceCode2Pdf.PythonScriptsExecution
+{
+    public class PythonScriptsExecution<T>
+    {
+        public T Execute(string scriptFile)
+        {
+            ScriptEngine pythonScript = Python.CreateEngine();
+            ScriptScope scope = pythonScript.CreateScope();
+            pythonScript.ExecuteFile(scriptFile);
+            return scope.GetVariable("result");// изменить имя переменной в зависимости от ее имени в скрипте
```

Comment:
if the result of script is store in some python variable than its name must be in input of method